src/services/TrackingService.ts

```diff
@@ -176,6 +176,7 @@ const getTotalPlantsInZones = async (organizationId: number, siteId: number): Pr
       'plantingSubzones.populations.species_scientificName',
       'plantingSubzones.populations.species_organization_id',
       'plantingSubzones.populations.totalPlants(raw)',
+      'plantingSubzones.populations.totalPlants',
```

Comment:
Added this back for the v1 plants dashboard map tooltip which renders totalPlants.

Reply:
Oops, I must have missed that one.